migration-guides/v6.0.0.md

```diff
@@ -0,0 +1,4 @@
+# v6.0.0 Migration Guide
+
+This version introduces support for [Node](https://nodejs.org/en/) version of ^16.0.0. No `lp-components` interface changes were introduced in this version, though many of the underlying dependencies used to build the `lp-components` have been updated. This version also eliminates support for earlier Node versions. [Client template](https://github.com/LaunchPadLab/client-template) version ^6.0.0 is compatible with this `lp-components` version.
+
```

Comment:
Let me know if you think there's more to say here.

Reply:
I think this needs to get tweaked slightly now to accommodate the ^14 allowance

Reply:
Good catch! Updated